bsip-0022.md

```diff
@@ -9,113 +9,439 @@
+
+### <div id="peerplays-vote-decay" /> Vote Decay on Peerplays
+
+A [proposal for Peerplays](https://github.com/peerplays-network/pips/blob/master/pip-0002.md) introduced vote decay as an element of its Gamified Proof of Stake (GPOS).  Voters receive rewards from a portion of the operation fees and "rakes" that are collected by the blockchain.  The reward is contingent on [the amount of stake that is vested by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1072-1080), and [the age of last casted vote by the voter](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L726-791).  The age of the last casted vote [also decays the voting power](https://github.com/peerplays-network/peerplays/blob/78dcc7df7316a0fcae8c14b7b648b88caa956326/libraries/chain/db_maint.cpp#L1446-1447) for all referendums on the platform.
+
 
 # Specifications
 
-## Committee controlled variables
+## <div id="referendum-categories" /> Referendum Categories
+
+BitShares has three categories of referendums that core token holders may vote on: witnesses, committee members, and worker proposals.  Prior to this proposal, any vote within any category applies until changed or until the proxy assignment had been assigned.  The retraction of proxy delegation re-instates a core token holders' prior voting selection for witnesses, committee members, and worker proposals.
+
+This proposal shall retain the ability of every core token holder to vote on all referendum categories and the optional delegation to a voting proxy.
+
+Votes for two of the referendum categories shall be "decayable": witnesses and committee members.  [Votes for worker proposals shall not be decayable](#out-of-scope) and shall continue to be tabulated as they were prior to this proposal because they are generally fixed in duration.  [Vote delegation to a proxy shall also be decayable](#proxy-power).
+
+
+## <div id="account-voting-stake" /> Voting Stake of an Account
+
+The full power of an account's vote equals the [the sum of](https://github.com/bitshares/bitshares-core/blob/bf7ff54d9a17aa43f4663521e371b8c0ddfc2284/libraries/chain/db_maint.cpp#L1184-1186) an account's liquid core tokens (BTS), core tokens in vesting balances, and core tokens in open orders including [collateral that is used for shorting smart assets into existence](https://github.com/bitshares/bitshares-core/blob/bf7ff54d9a17aa43f4663521e371b8c0ddfc2284/libraries/chain/market_evaluator.cpp#L234-245).
+
+_The full power of an account's vote will automatically be updated to reflect any changes to the liquid balance, vesting balance, or core tokens in open orders_.
```

Comment:
Perhaps add a note that this list may be expanded on by future BSIPs, e. g. Margin Trading.
The important sentence here is the next one: current calculation method shall be retained.